swap/src/bitcoin/punish.rs

```diff
@@ -1,12 +1,16 @@
 use crate::bitcoin::wallet::Watchable;
-use crate::bitcoin::{self, Address, PunishTimelock, Transaction, TxCancel, Txid};
+use crate::bitcoin::{self, Address, Amount, PunishTimelock, Transaction, TxCancel, Txid};
 use ::bitcoin::util::bip143::SigHashCache;
 use ::bitcoin::{SigHash, SigHashType};
 use anyhow::{Context, Result};
 use bdk::bitcoin::Script;
 use miniscript::{Descriptor, DescriptorTrait};
 use std::collections::HashMap;
 
+// Taken from https://mempool.space/testnet/tx/ed4d60bc1fd172feca444ed3d06cccb90346b9098c2d28d2d034dac66f608f68
+// The weight might fluctuate slightly in reality.
```

Comment:
❓ I find this comment a bit confusing - how would the weight of a transaction fluctuate? 
I thought the weight should always be the same, assuming that we always construct the transaction in the same way.

Reply:
>  I thought the weight should always be the same, assuming that we always construct the transaction in the same way.

That's the point: unfortunately we don't construct our transactions always in the same way. Every byte counts, e.g. a bigger amount, a different timeouts, etc. maybe even spending from a transaction on `vout=0` vs `vout=1` makes a difference :) 

Anyway, I had a chat with @thomaseizinger about it and these constants will go away. 
Wait for these changes please.